Code/Framework/AzToolsFramework/AzToolsFramework/Editor/RichTextHighlighter.cpp

```diff
@@ -29,11 +29,8 @@ namespace AzToolsFramework
         return highlightedString;
     }
 
-    void RichTextHighlighter::PaintHighlightedRichText(const QString& highlightedString,QPainter* painter, QStyleOptionViewItem option, QRect availableRect)
+    void RichTextHighlighter::PaintHighlightedRichText(const QString& highlightedString,QPainter* painter, QStyleOptionViewItem option, QRect availableRect, QPoint offset /* = QPoint()*/)
     {
-        painter->save();
-        painter->setRenderHint(QPainter::Antialiasing);
```

Comment:
The `painter` save/restore still need to happen inside this method because it modifies the `painter`